yt_dlp/extractor/newgrounds.py

```diff
@@ -202,7 +213,57 @@ def _real_extract(self, url):
                 continue
             entries.append(
                 self.url_result(
-                    'https://www.newgrounds.com/%s' % path,
+                    f'https://www.newgrounds.com/{path}',
                     ie=NewgroundsIE.ie_key(), video_id=media_id))
 
         return self.playlist_result(entries, playlist_id, title)
+
+
+class NewgroundsUserIE(InfoExtractor):
+    _VALID_URL = r'https?://(?P<id>[^\.]+)\.newgrounds\.com/(movies|audio)'
```

Comment:
```suggestion
    _VALID_URL = r'https?://(?P<id>[^\.]+)\.newgrounds\.com/(?:movies|audio)/?(?:[#?]|$)'
```
should fix the tests